### DISCUSSION TITLE
Release v0.9.4 / 5.51.4

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1131] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This is a maintenance release, fixing a few bugs and improving the usability with various small new features.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# Changelog

### added
- added clear commands to log sub menus [#391](https://github.com/sandboxie-plus/Sandboxie/issues/391)
- added option to disable prompt to confirm process termination [#514](https://github.com/sandboxie-plus/Sandboxie/issues/514)
- added settings to sandboxie-plus.ini "Options/InstantRecovery" to use the recovery window instad of the notification popUp [#988](https://github.com/sandboxie-plus/Sandboxie/issues/988)
- added ability to rename a non empty sandbox [#1100](https://github.com/sandboxie-plus/Sandboxie/issues/1100)
- added ability to remove a non empty sandbox 
- added file browser window to sandman ui to cover the files view functionality of sbie ctrl [#578](https://github.com/sandboxie-plus/Sandboxie/issues/578)

### Changed
- generic error in sbie ui now show the status code as hex and provide a string description when available

### fixed
- "del" shortcut to terminate a process did not always work
- fixed group display issue [#1094](https://github.com/sandboxie-plus/Sandboxie/issues/1094)
- fixed issue when using run sandboxed on a file located already in a sandbox [#1099](https://github.com/sandboxie-plus/Sandboxie/issues/1099)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/0.9.4'>Release v0.9.4 / 5.51.4</a>.</em>

**Comment by BilleBarrett** [comment]
I have just installed this update on 2 of my clients and all I get is a White screen 
This is happenned for Edge and Chrome

**Reply by isaak654** [reply]
I can't reproduce it, is this happening on new created sandboxes as well?

**Comment by BilleBarrett** [comment]
I just installed it on 1 remote client and then I have a system here on my work bench 

On the remote I uninstalled the update and removed Sadnboxie.ini and reinstalled the Sandboxie 5.51.3 

As soon as I turn on Force Chrome to run in this sandbox 

Now I get the error  SBIE224 Sandboxed program has crashed: Chrome.exe default

Sandboxie.ini

```
[Template_Chrome_History_DirectAccess]

Tmpl.Title=#4336,Google Chrome

Tmpl.Class=WebBrowser

OpenFilePath=chrome.exe,%Tmpl.Chrome%\Bookmarks*

OpenFilePath=chrome.exe,%Tmpl.Chrome%\Favicons*

OpenFilePath=chrome.exe,%Tmpl.Chrome%\*History*

OpenFilePath=chrome.exe,%Tmpl.Chrome%\Current *

OpenFilePath=chrome.exe,%Tmpl.Chrome%\Last *

OpenFilePath=chrome.exe,%Tmpl.Chrome%\Visited Links*


[GlobalSettings]

 

Template=RpcPortBindings

Template=WindowsRasMan

Template=SynapticsTouchPad

Template=WindowsLive

Template=OfficeLicensing

Template=OfficeClickToRun

FileRootPath=C:\Sandbox\%USER%\%SANDBOX%

 

[DefaultBox]

 

ConfigLevel=9

Template=Edge_Force

Template=Chrome_Force

Template=WindowsDefender

Template=BitDefenderInternetSecurity

Template=AdobeDistiller

Template=AdobeAcrobatReader

Template=AdobeAcrobat

Template=Windows_Live_Mail

Template=Office_Outlook

Template=Chrome_Profile_DirectAccess

Template=Chrome_Sync_DirectAccess

Template=Chrome_Preferences_DirectAccess

Template=Chrome_Passwords_DirectAccess

Template=Chrome_Cookies_DirectAccess

Template=Chrome_History_DirectAccess

Template=Chrome_Bookmarks_DirectAccess

Template=Edge_Profile_DirectAccess

Template=Edge_Phishing_DirectAccess

Template=Edge_Sync_DirectAccess

Template=Edge_Preferences_DirectAccess

Template=Edge_Passwords_DirectAccess

Template=Edge_Cookies_DirectAccess

Template=Edge_History_DirectAccess

Template=Edge_Bookmarks_DirectAccess

Template=OpenSmartCard

Template=OpenBluetooth

Template=SkipHook

Template=FileCopy

Template=qWave

Template=BlockPorts

Template=LingerPrograms

Template=Chrome_Phishing_DirectAccess

Template=Firefox_Phishing_DirectAccess

Template=AutoRecoverIgnore

RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%

RecoverFolder=%Personal%

RecoverFolder=%Desktop%

BorderColor=#00FF00,ttl,6

Enabled=y

BoxNameTitle=y

AutoDelete=y

NeverDelete=n

CopyLimitKb=249152

ProcessGroup=<Chrome>,chrome.exe,msedge.exe

 

[UserSettings_21B2036F]

 

SbieCtrl_UserName=dick_gray

SbieCtrl_BoxExpandedView=DefaultBox

SbieCtrl_NextUpdateCheck=1630415744

SbieCtrl_ReloadConfNotify=n

SbieCtrl_EditConfNotify=n

SbieCtrl_AutoApplySettings=y

SbieCtrl_SettingChangeNotify=n

SbieCtrl_TerminateWarn=n

SbieCtrl_HideWindowNotify=n

SbieCtrl_WindowCoords=508,193,1237,607

SbieCtrl_ActiveView=40021

SbieCtrl_ProcessViewColumnWidths=250,70,300

 

[GlobalSettings]

 

Template=RpcPortBindings

Template=WindowsRasMan

Template=SynapticsTouchPad

Template=WindowsLive

Template=OfficeLicensing

Template=OfficeClickToRun

FileRootPath=C:\Sandbox\%USER%\%SANDBOX%

 

[DefaultBox]

 

ConfigLevel=9

Template=Edge_Force

Template=Chrome_Force

Template=WindowsDefender

Template=BitDefenderInternetSecurity

Template=AdobeDistiller

Template=AdobeAcrobatReader

Template=AdobeAcrobat

Template=Windows_Live_Mail

Template=Office_Outlook

Template=Chrome_Profile_DirectAccess

Template=Chrome_Sync_DirectAccess

Template=Chrome_Preferences_DirectAccess

Template=Chrome_Passwords_DirectAccess

Template=Chrome_Cookies_DirectAccess

Template=Chrome_History_DirectAccess

Template=Chrome_Bookmarks_DirectAccess

Template=Edge_Profile_DirectAccess

Template=Edge_Phishing_DirectAccess

Template=Edge_Sync_DirectAccess

Template=Edge_Preferences_DirectAccess

Template=Edge_Passwords_DirectAccess

Template=Edge_Cookies_DirectAccess

Template=Edge_History_DirectAccess

Template=Edge_Bookmarks_DirectAccess

Template=OpenSmartCard

Template=OpenBluetooth

Template=SkipHook

Template=FileCopy

Template=qWave

Template=BlockPorts

Template=LingerPrograms

Template=Chrome_Phishing_DirectAccess

Template=Firefox_Phishing_DirectAccess

Template=AutoRecoverIgnore

RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%

RecoverFolder=%Personal%

RecoverFolder=%Desktop%

BorderColor=#00FF00,ttl,6

Enabled=y

BoxNameTitle=y

AutoDelete=y

NeverDelete=n

CopyLimitKb=249152

ProcessGroup=<Chrome>,chrome.exe,msedge.exe

 

[UserSettings_21B2036F]

 

SbieCtrl_UserName=dick_gray

SbieCtrl_BoxExpandedView=DefaultBox

SbieCtrl_NextUpdateCheck=1630415744

SbieCtrl_ReloadConfNotify=n

SbieCtrl_EditConfNotify=n

SbieCtrl_AutoApplySettings=y

SbieCtrl_SettingChangeNotify=n

SbieCtrl_TerminateWarn=n

SbieCtrl_HideWindowNotify=n

SbieCtrl_WindowCoords=508,193,1237,607

SbieCtrl_ActiveView=40021

SbieCtrl_ProcessViewColumnWidths=250,70,300
```



 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Comment by BilleBarrett** [comment]
When I removed Force into the Sandbox everting is ok

The instant I turn on the force into the sandbox I get a white screen



 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Reply by isaak654** [reply]
> 
> When I removed Force into the Sandbox everting is ok The instant I turn on the force into the sandbox I get a white screen

Unfortunately I wasn't able to reproduce your issue with `Template=Edge_Force`.
 
You could have a mismatch between the Chrome version you have outside of the sandbox and the Chrome version you have inside the sandbox... did you try to create a new sandbox with the force setting?

**Comment by BilleBarrett** [comment]
OK so IN added Firefox, 1st I could not start it with Sandboxie on [it said it had not profile], ran outside Sandboxie and it start up and then ran it in Sandboxie and it was OK

Still no chrome or Edge
If you want see this computer I will gladly make it available to you

 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Comment by BilleBarrett** [comment]
There is no difference between Chrome, I just checked
I uninstalled Sandboxie all version from the client computer

I removed the Sadnboxie.ini file from Windows
I restart the computer and then installed Sandboxie 5.51.4 and edited the default settings

Again as soon as I turn on the force it get the white screen, it seems to have not effect on Firefox

 

 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Reply by isaak654** [reply]
It looks like you have two `[DefaultBox]` and two `[Usersettings]` on your ini file, I noticed similar bugs in previous 0.9.x pre-releases.

Another thing that you could try is to remove this line, it seems unused: `ProcessGroup=<Chrome>,chrome.exe,msedge.exe`

**Comment by BilleBarrett** [comment]
I have uninstall Sandboxie and the delete the Ini file, then I installed Sandboxie and did a complete setup 

Why does Firefox work?  i.e. it is not Chrome based

 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Reply by isaak654** [reply]
Firefox has a different folder structure and template rules than Chrome.. however, no change was made on 5.51.4 for both Chrome & Firefox templates.

**Comment by BilleBarrett** [comment]
I have totally uninstalled Sandboxie from my clients system, ran registry cleaner, deleted the Sandboxie folders 

Now I’m rebooting his computer

Then I will install Sandboxie 5.51.3 1st and set it up, mid you \this has been funning Sandboxie for well over a year with updates

 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Comment by BilleBarrett** [comment]
Restarted and installed Sandboxie 5.51.3, make no changes to the settings

1.	I did a Microsoft Edge repair, which download a new copy of Edge
2.	I uninstalled Google Chrome 
3.	Right click on Firefox open in the Sandbox without any problem
4.	Right clicked Microsoft edge and told it to run in Sandboxie [white screen again, could see it try to download stuff]
5.	Download Chrome 92.0.4515.159 [latest]
6.	Right click on installed Chrome then right clicked on it and told it to run Sandboxed [White screen]

I cannot find anything I have gone thru the registry, nothing 

 

 

 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Reply by isaak654** [reply]
Another possibility is a sudden third-party conflict... for example BitDefender caused issues with Sandboxie in past releases and a specific workaround was needed: https://github.com/sandboxie-plus/Sandboxie/issues/715#issuecomment-822989295

Besides this, I'm out of ideas.

**Comment by BilleBarrett** [comment]
Bitdefender is my endpoint I use to monitor clients, I will removed it

 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Comment by BilleBarrett** [comment]
My big problem is I have over 200 monitoring clients all using Bitdefender ☹

 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Comment by DavidXanatos** [comment]
First of all is it confirmed that its a bit deffender problem?
second if you go back to the old version does the issue go away?

**Comment by BilleBarrett** [comment]
I am working on that right now

 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Comment by BilleBarrett** [comment]
I just check one of my clients that I installed 5.51.4 on yesterday, run Bitdefender and everything is Ok.

Going back to the Lenovo and see what is going on



 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Comment by BilleBarrett** [comment]
It seems this client has some setting he must have screwed around with and messed up the intel 4600 graphics, still working on it

 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Comment by BilleBarrett** [comment]
I’m not sure what is happening, but I did install the update on another client and it took with out a problem.
This other Lenovo has either a video problem, [loaded new drivers] or a bad memory 

Sorry

 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Comment by BilleBarrett** [comment]
This very confusing, when I right click on Google and tell it to run outside of Sandboxie everything is OK, but tell it to fun Sandboxed it get a white screen

Any ideas,

Ran all the diagnostics, 

Bill

 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Comment by BilleBarrett** [comment]
Why do I get this error constantly

SBIE2211 Sandboxed service failed to start:  BITS

SBIE2211 Sandboxed service failed to start:  BITS

Windows 11 Pro 

On Chrome and Microsoft Edge [chrome]

 

 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Reply by isaak654** [reply]
> Why do I get this error constantly SBIE2211 Sandboxed service failed to start: BITS SBIE2211 Sandboxed service failed to start: BITS Windows 11 Pro On Chrome and Microsoft Edge [chrome] 

See https://github.com/sandboxie-plus/Sandboxie/issues/1081

**Comment by BilleBarrett** [comment]
If I check to Drop Admin I lose the internet, turn it on and I’m OK

I have been a Sandboxie user since version 2 and I own several; licenses, I just want it to work for my clients 

I switched over to the Plus version which I have been running since it was released

I running 

Google Chrome is up to date

Version 92.0.4515.159 (Official Build) (64-bit)

Microsoft Edge Version 92.0.902.78 (Official build) (64-bit)



If I drop Admin Google & Microsoft will not open


Start Chrome and get this  and nothing loads

Microsoft shows this


And this 

Now I did thios



Chrome and Microsoft Chrome opens with any problem

Still getting errors:

Sadnboxie.ini

[GlobalSettings]

Template=RoboForm

Template=RpcPortBindings

Template=WindowsRasMan

Template=WindowsLive

Template=KeyScrambler

Template=OfficeLicensing

Template=OfficeClickToRun

FileRootPath=C:\Sandbox\%USER%\%SANDBOX%

SeparateUserFolders=y

KeyRootPath=\REGISTRY\USER\Sandbox_%USER%_%SANDBOX%

IpcRootPath=\Sandbox\%USER%\%SANDBOX%\Session_%SESSION%

EditAdminOnly=n

ForceDisableAdminOnly=n

ForgetPassword=n

NetworkEnableWFP=y

StartRunAlertDenied=y

NotifyStartRunAccessDenied=y

AlertFolder=C:\Users\BilleB\AppData\Local\Take Control Viewer

 

[DefaultBox]

 

ConfigLevel=9

Template=nVidia_nView

Template=Edge_Phishing_DirectAccess

Template=Firefox_Profile_DirectAccess

Template=Firefox_Session_DirectAccess

Template=Firefox_Phishing_DirectAccess

Template=Firefox_Passwords_DirectAccess

Template=Firefox_Cookies_DirectAccess

Template=Firefox_Bookmarks_DirectAccess

Template=Firefox_Force

Template=OpenBluetooth

Template=WindowsDefender

Template=AdobeAcrobatReader

Template=AdobeAcrobat

Template=Office_Outlook

Template=Chrome_Profile_DirectAccess

Template=Chrome_Sync_DirectAccess

Template=Chrome_Preferences_DirectAccess

Template=Chrome_Passwords_DirectAccess

Template=Chrome_Cookies_DirectAccess

Template=Chrome_History_DirectAccess

Template=Chrome_Bookmarks_DirectAccess

Template=Chrome_Force

Template=Edge_Profile_DirectAccess

Template=Edge_Sync_DirectAccess

Template=Edge_Preferences_DirectAccess

Template=Edge_Passwords_DirectAccess

Template=Edge_Cookies_DirectAccess

Template=Edge_History_DirectAccess

Template=Edge_Bookmarks_DirectAccess

Template=Edge_Force

Template=SkipHook

Template=FileCopy

Template=qWave

Template=BlockPorts

Template=LingerPrograms

Template=Chrome_Phishing_DirectAccess

Template=AutoRecoverIgnore

RecoverFolder=M:\Downloads

BorderColor=#00ff00,ttl,6

Enabled=y,Administrators,BilleB

BoxNameTitle=y

AutoDelete=y

AutoRecover=y

ForceFolder=W:\

CopyLimitKb=249152

OpenPrintSpooler=y

AllowSpoolerPrintToFile=y

AllowRawDiskRead=y

NotifyDirectDiskAccess=y

ProcessGroup=<ChromePrograms>,vivaldi.exe,opera.exe,msedge.exe,Maxthon.exe,iron.exe,dragon.exe,chrome.exe,browser.exe,brave.exe,acrord32.exe

ProcessGroup=<Other Program>,takecontrolrdviewer.exe,takecontrolrdldr.exe,acrord32.exe

ClosedFilePath=<BlockNetDevices>,InternetAccessDevices

AllowNetworkAccess=<BlockNetAccess>,n

PromptForInternetAccess=y

MsiInstallerExemptions=y

NotifyInternetAccessDenied=y

 

[UserSettings_4BC00582]

 

SbieCtrl_UserName=administrator

SbieCtrl_BoxExpandedView=DefaultBox

SbieCtrl_NextUpdateCheck=1618777607

SbieCtrl_AutoApplySettings=y

SbieCtrl_SettingChangeNotify=n

SbieCtrl_HideWindowNotify=n

SbieCtrl_WindowCoords=200,150,1237,632

SbieCtrl_ActiveView=40021

 

[UserSettings_10FE026B]

 

SbieCtrl_UserName=billeb

SbieCtrl_WindowCoords=221,113,1237,632

SbieCtrl_ActiveView=40021

SbieCtrl_NextUpdateCheck=1630333802

SbieCtrl_AutoApplySettings=y

SbieCtrl_SettingChangeNotify=n

SbieCtrl_HideWindowNotify=n

SbieCtrl_RecoverTarget=M:\1A-4-USB's\1A_latest updates

SbieCtrl_RecoverTarget=M:\PatchMyPc

SbieCtrl_RecoverTarget=M:\1A-4-USB's\1-CPI-ID PROGRAMS\cpu-z-en

SbieCtrl_RecoverTarget=M:\1A-4-USB's\1-CPI-ID PROGRAMS\System Ninja

SbieCtrl_RecoverTarget=M:\1A-4-USB's\1-CPI-ID PROGRAMS\hw64

SbieCtrl_SaveRecoverTargets=y

SbieCtrl_TerminateWarn=n

SbieCtrl_ProcessViewColumnWidths=250,70,300

SbieCtrl_AutoStartAgent=SandMan.exe

SbieCtrl_EnableAutoStart=y

SbieCtrl_UpdateCheckNotify=n

SbieCtrl_BoxExpandedView=DefaultBox

 

[UserSettings]

 

SbieCtrl_AutoStartAgent=SandMan.exe

AutoRecover=n

BlockNetworkFiles=y

RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%

RecoverFolder=%Personal%

RecoverFolder=%Desktop%

BorderColor=#00FFFF,ttl

Template=OpenBluetooth

Template=SkipHook

Template=FileCopy

Template=qWave

Template=BlockPorts

Template=LingerPrograms

Template=Chrome_Phishing_DirectAccess

Template=Firefox_Phishing_DirectAccess

Template=AutoRecoverIgnore

ConfigLevel=9

SbieCtrl_EnableAutoStart=y



 

 





Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Reply by isaak654** [reply]
It looks like you tried to post some images, but I can't see any of them:
https://user-images.githubusercontent.com/12372772/131039790-3ffc240c-50a2-4533-ac8f-88ede2973f7b.png

**Comment by BilleBarrett** [comment]
I’ll just leave it be and put up with messages, until someone can figure out shy it doing it.

It is a security failure I found in my event log
An account failed to log on.

Subject:

                Security ID:                         BCS-COREI7\BilleB

                Account Name:                 BilleB

                Account Domain:                              BCS-COREI7

                Logon ID:                             0x1501F8

Logon Type:                                       5

Account For Which Logon Failed:

                Security ID:                         NULL SID

                Account Name:                 -

                Account Domain:                              -

Failure Information:

                Failure Reason:                 An Error occurred during Logon.

                Status:                                  0xC0000022

                Sub Status:                         0xC0000022

Process Information:

                Caller Process ID:              0x25e8

                Caller Process Name:      C:\Program Files\Sandboxie-Plus\SandboxieBITS.exe

Network Information:

                Workstation Name:         -

                Source Network Address:             -

                Source Port:                       -

Detailed Authentication Information:

                Logon Process:                  Advapi  

                Authentication Package:               Negotiate

                Transited Services:          -

                Package Name (NTLM only):        -

                Key Length:                        0

This event is generated when a logon request fails. It is generated on the computer where access was attempted.

 

The Subject fields indicate the account on the local system which requested the logon. This is most commonly a service such as the Server service, or a local process such as Winlogon.exe or Services.exe.

The Logon Type field indicates the kind of logon that was requested. The most common types are 2 (interactive) and 3 (network).

The Process Information fields indicate which account and process on the system requested the logon.

The Network Information fields indicate where a remote logon request originated. Workstation name is not always available and may be left blank in some cases.

The authentication information fields provide detailed information about this specific logon request.

                - Transited services indicate which intermediate services have participated in this logon request.

                - Package name indicates which sub-protocol was used among the NTLM protocols.

                - Key length indicates the length of the generated session key. This will be 0 if no session key was requested.

From my system


The BITS service failed to start.  Error 0x80070005.

DCOM got error "1069" attempting to start the service BITS with arguments "Unavailable" in order to run the server:

{4991D34B-80A1-4291-83B6-3328366B9097}

 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Reply by isaak654** [reply]
I've found a workaround to disable BITS, please look at this post and report if it works: https://github.com/sandboxie-plus/Sandboxie/issues/437#issuecomment-782597703

**Comment by BilleBarrett** [comment]
I’m not sure where and how to edit tis sbie.ini

Not sure where it is

Can you post a description a little better

Please try adding ClosedClsid={4991D34B-80A1-4291-83B6-3328366B9097} to your sbie ini, it disable BITS as this is thoroughly broken in win 10 when sandboxed

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Reply by isaak654** [reply]
Assuming you're on the main Sandboxie Plus window and you've closed any running program in the sandbox:
1) Go to Options menu -> Edit ini file -> confirm yes
2) Add `ClosedClsid={4991D34B-80A1-4291-83B6-3328366B9097}` below [DefaultBox]
3) Save the ini file and close it
4) Go to Options menu -> Reload ini file

**Comment by BilleBarrett** [comment]
Ok, I did test tomorrow
